### DISCUSSION TITLE
请问PCL2编译的时候是怎么把DLL合并进EXE的啊

### DISCUSSION THREAD

**Discussion by SALTWOOD** [Meloong-Git/PCL#2569] [提问] [answered]
看了PCL2的项目文件，似乎没用到Fody之类用于合并lib的NuGet包，请问PCL2发布的时候是龙猫手动合并dll的吗
我编译出的程序必须和那几个dll放在一起才能用
网上也找不到好的方法

**Comment by MoYuan-CN** [comment]
~~参考 https://blog.csdn.net/Carry_Lee/article/details/102457495 ?~~
EDIT：见下

**Reply by SALTWOOD** [reply]
似乎没用（？）而且这是个NuGet包

**Comment by LTCatt** [comment, marked as answer]
Application.xaml.vb

**Comment by Yue-cn** [comment]
用Fody,（他不好吗？）

**Reply by MoYuan-CN** [reply]
Fody 好（刚看到了）
